docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,15 +506,27 @@ If you submit a ledger API command (which can have multiple Daml commands), then
 
 Just putting all Daml commands into a single batch command does not help, as you might still create lots of views.
 
+Starting from protocol version 6, Canton creates a view for every action node in the transaction tree if the participants
+that host their informees are not a subset of their parents' informee participants. Therefore, in order to
+minimize the number of views, you should try to minimize either the number of changes of informees or make sure
+most of your actions share the same pool of participants that must be informed. Another alternative is to
+make your higher level action nodes include as much participants as possible so that subsequent action nodes in the tree
+are aggregated in the initial view, since these children nodes will only target a subset of those participants.
+Informees are roughly the signatories, observers of the contract and of the choice, and the controllers of the choice.
+
 In the protocol up to version 5, Canton creates a view for every action node in the transaction tree if the informees change 
-between the node and its parent. Therefore, in order to minimize the number of views, you should try to minimize the number of 
-changes of informees. Informees are roughly the signatories, observers of the contract and of the choice, and the controllers of the choice. 
+between the node and its parent. Therefore, in this case, in order to minimize the number of views, your only alternative
+is to try to minimize the number of changes of informees.
 
-Instead of sending a batch with 20 exercise commands `Foo`, group your contracts by stakeholder and send one exercise 
-command on a generator contract with the list of contracts that in turn exercises the `Foo` choice per stakeholder group.
+Instead of sending a batch with 20 exercise commands `Foo`, group your contracts by a set of stakeholders
+that cover the exercise's informee participants, or a single stakeholder if your are using protocol version
+5 or older. You then send one exercise command on a generator contract with the list of contracts that in
+turn exercises the `Foo` choice per stakeholder group.
 
-If you write batch commands, group the operations on the ledger based on the informees and create a single choice and 
-batch operation per informee group. You can use choice observers to align informees if necessary. 
+If you write batch commands, group the operations on the ledger based on the informees and their hosting participants
+and create a single choice and batch operation per informee's participant group.
+For older protocol versions (v5<) that only take in consideration the informees and not their hosting participants, you can
+use choice observers to align informees if necessary.
```

Comment:
I think this test would be easier to follow if you introduce two sections here: One for PV <= 5 and one for PV 6 and greater. There may be a bit of repetition between the two sections, but readers should know what protocol version they are targeting and can then focus on that section. As is, they always have to evaluate these conditionals in their head in every sentence.